drivers/wifi/nrf700x/src/wifi_mgmt.c

```diff
@@ -739,14 +739,24 @@ int nrf_wifi_mode(const struct device *dev,
 
 	} else {
 		mode->mode = def_dev_ctx->vif_ctx[vif_ctx_zep->vif_idx]->mode;
+
+		/**
+		 * This is a work-around to maintain umac context for mode same as much
```

Comment:
Do we need to mention that this is a `workaround` here now...as in is there a plan to `fix` this in the future? If not then it might be better to remove the `workaround` term here.

Reply:
I do not think there is any plan to change this. I will remove this